maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -397,10 +397,10 @@ void cli( CliRequest cliRequest )
                 }
 
                 mavenConfig = cliManager.parse( args.toArray( new String[0] ) );
-                List<?> unrecongized = mavenConfig.getArgList();
-                if ( !unrecongized.isEmpty() )
+                List<?> unrecognized = mavenConfig.getArgList();
+                if ( !unrecognized.isEmpty() )
                 {
-                    throw new ParseException( "Unrecognized maven.config entries: " + unrecongized );
+                    throw new ParseException( "Unrecognized maven.config entries: " + unrecognized );
```

Comment:
This typo fix is not directly related to the fix anymore, so I can leave it out of this PR, if needed.